### DISCUSSION TITLE
Changing enum values

### DISCUSSION THREAD

**Discussion by danmoser** [FreeOpcUa/opcua-asyncio#853] [Q&A] [answered]
Hi there?
Could someone tell how to change `tutu` value from 2 to 10 in the example below?
Thanks in advance!

```
    enode = await new_enum(server, idx, "MyEnum", [
        "titi",
        "toto",
        "tutu",
    ])
```

**Comment by AndreasHeine** [comment, marked as answer]
not with a enum! no gaps!

probably a MultiStateValueDiscreteType:
https://reference.opcfoundation.org/Core/docs/Part8/5.3.3/#5.3.3.4

EnumValues is an array of EnumValueType. Each entry of the array represents one enumeration value with its integer notation, a human-readable representation, and help information. This represents enumerations with integers that are not zero-based or have gaps (e.g. 1, 2, 4, 8, 16). See [OPC 10000-3](https://reference.opcfoundation.org/v105/Core/docs/Part3/) for the definition of this type. MultiStateValueDiscrete Variables expose the current integer notation in their Value Attribute. Clients will often read the EnumValues Property in advance and cache it to lookup a name or help whenever they receive the numeric representation.
